### PR TITLE
[nokia] Replace os.system and remove subprocess with shell=True

### PR DESCRIPTION
**Pull request by maipbui**
Signed-off-by: maipbui <maibui@microsoft.com>

Dependency: [https://github.com/sonic-net/sonic-buildimage/pull/12065](https://github.com/sonic-net/sonic-buildimage/pull/12065)
#### Why I did it
`subprocess.Popen()` and `subprocess.run()` is used with `shell=True`, which is very dangerous for shell injection.
`os` - not secure against maliciously constructed input and dangerous if used to evaluate dynamic content
`getstatusoutput` is dangerous because it contains `shell=True` in the implementation
#### How I did it
Replace `os` by `subprocess`, use with `shell=False`
Remove unused functions
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [X] 201911
- [X] 202006
- [X] 202012
- [X] 202106
- [X] 202111
- [X] 202205

### CODE REVIEW

File: device/nokia/armhf-nokia_ixs7215_52x-r0/plugins/eeprom.py [hunk 1]
```diff
@@ -10,5 +10,7 @@ class board(eeprom_tlvinfo.TlvInfoDecoder):
     def __init__(self, name, path, cpld_root, ro):
         self.eeprom_path = "/sys/class/i2c-adapter/i2c-0/0-0053/eeprom"
         if not os.path.exists(self.eeprom_path):
-            os.system("echo 24c02 0x53 > /sys/class/i2c-adapter/i2c-0/new_device")
+            file = "/sys/class/i2c-adapter/i2c-0/new_device"
+            with open(file, 'w') as f:
+                f.write('24c02 0x53')
```
Comment (qiluo-msft) [inline device/nokia/armhf-nokia_ixs7215_52x-r0/plugins/eeprom.py:15]: >0x53 [](http://example.com/codeflow?start=31&length=4)

Did you miss `\n`?

File: platform/marvell-armhf/sonic-platform-nokia/7215/sonic_platform/chassis.py [hunk 2]
```diff
@@ -331,7 +327,7 @@ def set_status_led(self, color):
 
         # Write sys led
         if smbus_present == 0:  # called from host (e.g. 'show system-health')
-            cmdstatus, value = cmd.getstatusoutput('sudo i2cset -y 0 0x41 0x7 %d' % value)
+            cmdstatus, value = getstatusoutput_noshell(['sudo', 'i2cset', '-y', '0', '0x41', '0x7', value])
```
Comment (qiluo-msft) [inline platform/marvell-armhf/sonic-platform-nokia/7215/sonic_platform/chassis.py:330]: >value [](http://example.com/codeflow?start=100&length=5)

`str(value)`.

**Review by qiluo-msft — APPROVED** [review]

**Comment by maipbui** [comment]
@carl-nokia @dflynn-Nokia Could you help review and verify?

**Comment by maipbui** [comment]
@mlok-nokia Could you help review and verify?

**Comment by qiluo-msft** [comment]
@Blueve is connecting Nokia reviewers.

**Comment by mlok-nokia** [comment]
I am ok with the change. Thanks

**Pull request merged**